### ISSUE TITLE
Create Invitation API request

### ISSUE THREAD

**Issue by daniel-simpson** [Azure/azure-sdk-for-go#4086]
### Service Package Request

Very interested in support for the MS Graph API [invitations endpoint](https://docs.microsoft.com/en-us/graph/api/invitation-post?view=graph-rest-1.0).  My apologies if I'm missing an existing implementation, but I'm unable to see any reference to this API within the source of this repo.

I would like to be able to use this invitation endpoint (along with the already implemented GetUser and DeleteUser endpoints) to enhance the terraform AzureAD provider [as per this Pull Request](https://github.com/terraform-providers/terraform-provider-azuread/issues/41)

Any assistance would be greatly appreciated.  If need be I'm happy to have a hand in implementing, but I'm still quite new to Go and wasn't able to figure out the codegen process.

Thank you!

**Comment by jhendrixMSFT** [comment]
@joshgav I don't see a swagger for this API, can you please contact the endpoint owner to see if they have a plan for creating one?

**Comment by dansanabria** [comment]
Did you ever get a reply from the endpoint owner @daniel-simpson ?

**Comment by daniel-simpson** [comment]
@dansanabria unfortunately not, and as I was only doing investigation at that point we ended up having to park this piece of work :(

**Comment by mwieczorek** [comment]
@jhendrixMSFT I'd like to get this in the sdk too. 
You suggested to `contact the endpoint owner` - how can I do that? I mean: how can I find the endpoint owner?

**Comment by jhendrixMSFT** [comment]
@maggiepint do you know who to follow up with about this?

**Comment by Maarc-D** [comment]
Hello,

Same need, I've try to find out where adding config file to let autorest mais the job of the services folder like said in https://github.com/Azure/azure-sdk-for-go/blob/master/CONTRIBUTING.md 
I tried to start implementation on : https://github.com/Maarc-D/azure-sdk-for-go/tree/feat/graph-api-1.0 Because it's really needed on our side 
But I can't find out where a stored the configfiles for autorest

Someone can tell me what I missed plz ?

like @daniel-simpson the aim is to create the autorest config matchin microsoft api graph : https://docs.microsoft.com/en-us/graph/api/overview?view=graph-rest-1.0

Regards Marc.

**Comment by Maarc-D** [comment]
Hello,

Does someone know if I need to staticly put the go code generated by the swagger I created : https://github.com/Azure/azure-rest-api-specs/pull/10275

or does it will be automatic once merged ?

because I've test all the stack using the generated go files to code the missing feature on terraform provider : https://github.com/terraform-providers/terraform-provider-azuread/pull/297

it's working fine

Regards Marc.

**Comment by ArcturusZhang** [comment]
Hi @Maarc-D once this swagger PR is merged, you could just open a new issue using the feature-request template, and then I will handle the release process.

But I am curious that would they approve a new swagger authored by outside of the service team?

**Comment by lirenhe** [comment]
We don't have microsoft.graph in https://github.com/Azure/azure-rest-api-specs/.    The openAPI specs are maintained by MicrosoftGraph team in a separate repo https://github.com/microsoftgraph/msgraph-metadata.  The GO SDK would not use the openapi specs in that repo. So if you still have this request, please contact msgraph team directly. Thanks
